File: java_work/ex0708/src/ex0708/Circle.java
package ex0708;

/*
 * 216p
 */
class Point {
	int xPos, yPos;

	public Point(int xPos, int yPos) {// 생성자
		this.xPos = xPos;
		this.yPos = yPos;
	}

	public void showPoinInfo() {
		System.out.println("[" + xPos + " , " + yPos + "]");
	}
}

public class Circle{

	Point point; //변수지정
	int r;

	public Circle(int i, int j, int k) {
		point = new Point(i, j);
		this.r = k;
	}

	private void showCircleInfo() {
		System.out.println("반지름 r = "+this.r);
		point.showPoinInfo();

	}

	public static void main(String[] args) {
		Circle circle = new Circle(2, 2, 4);
		circle.showCircleInfo();
	}

}

File: python_work/220708/circle.py
from point import Point

class Circle:
    #p = point() 생략가능
    def __init__(self,x,y,r):#생성자
        print("온다", x, y, r)
        self.p = Point(x,y)
        self.r = r


    def showInfo(self):
        self.p.showInfo()
        #pass #내용없을 때 적어줘야
        print(f"반지름 = {self.r}")

c =  Circle(2,2,4)
c.showInfo()
